1_lieutenant_on_minikube.sh

```diff
@@ -1,34 +1,40 @@
 #!/usr/bin/env bash
 
+# shellcheck disable=SC1091
 source lib/functions.sh
 source lib/minikube.sh
 
-check_variable "GITLAB_TOKEN" $GITLAB_TOKEN
-check_variable "GITLAB_ENDPOINT" $GITLAB_ENDPOINT
-check_variable "GITLAB_USERNAME" $GITLAB_USERNAME
+check_variable "GITLAB_TOKEN" "$GITLAB_TOKEN"
+check_variable "GITLAB_ENDPOINT" "$GITLAB_ENDPOINT"
+check_variable "GITLAB_USERNAME" "$GITLAB_USERNAME"
 
 # Minikube must be running
-minikube start --memory 4096 --disk-size 60g --cpus 4
+minikube start --disk-size 60g --cpus 4
```

Comment:
Minikube chokes on Btrfs-on-LUKS because the Kubelet in the docker container cannot find the device for the cryptvolume in `/dev/mapper`. The error is something like:

```
Apr 26 15:50:53 minikube kubelet[2315]: W0426 15:50:53.878466    2315 fs.go:565] stat failed on /dev/mapper/nvme0n1p2_crypt with error: no such file or direct
ory
Apr 26 15:50:53 minikube kubelet[2315]: F0426 15:50:53.878479    2315 kubelet.go:1350] Failed to start ContainerManager failed to get rootfs info: failed to g
et device for dir "/var/lib/kubelet": could not find device with major: 0, minor: 29 in cached partitions map
```

Some googling found the following issue comment which suggests disabling a feature gate on sufficiently recent K8s versions: https://github.com/kubernetes/minikube/issues/7923#issuecomment-737058797'

This fixed the problem for me.

```suggestion
minikube start --disk-size 60g --cpus 4 --feature-gates="LocalStorageCapacityIsolation=false"
```